src/github.com/getlantern/lantern-ui/app/js/services.js

```diff
@@ -209,15 +196,32 @@ angular.module('app.services', [])
     });
 
     function trackPageView(sessionControl) {
-      var page = model.modal || '/';
-      ga('set', 'page', page);
-      ga('send', 'pageview', sessionControl ? {sessionControl: sessionControl} : undefined);
-      //log.debug(sessionControl === 'end' ? 'sent analytics session end' : 'tracked pageview', 'page =', page);
+      var trackers = ga.getAll();
+      for (var i =0; i < trackers.length; i++) {
+          var tracker = trackers[i];
+          if (tracker.b && tracker.b.data && tracker.b.data.w) {
+              var fields = tracker.b.data.w;
+              var gaObj = {
+                  clientId: '',
+                  clientVersion: '',
+                  language: '',
+                  screenColors: '',
+                  screenResolution: '',
+                  trackingId: '',
+                  viewPortSize: ''
+              };
+              for (var name in fields) {
```

Comment:
Good idea. How does the client id differ from the visitor id? I noticed that id is the same across clients. Can we just use it?